### PR TITLE
- Resolves issue #16

### PR DESCRIPTION
**Pull request by VictorMirandaEE**
- Resolves issue #16
- Adjusted styling of map view for aid requests
- Adjusted styling of the hospital list of requests (donor view)

### CODE REVIEW

File: main/static/js/map.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+
+  function fitBounds () {
+    map.fitBounds(hospitalGroup.getBounds());
+  }
+
+  // fit hospitalGroup after 1 second.
+  setTimeout(function () {
+    //fitBounds();
+  }, 1);
```
Comment (fmarani) [inline main/static/js/map.js:104]: This can be removed

File: main/templates/main/hospital_filter.html [hunk 2]
```diff
@@ -3,75 +3,76 @@
...
-    </div>
-    <div class="p-5 my-2 rounded-full">
-      <a href="{% url 'aidrequestfordonor_list' %}">
-        <img src="{% static "images/list-white.svg" %}" alt="list">
-      </a>
-    </div>
-    <div class="p-5 my-2 rounded-full bg-white">
-      <img src="{% static "images/map-blue.svg"  %}" alt="map">
+  <span class="flex-grow whitespace-no-wrap self-center text-3xl font-bold">Aid requests</span>
+  <a class="flex flex-shrink-0 self-center" href="{% url 'aidrequestfordonor_list' %}">
```
Comment (fmarani) [inline main/templates/main/hospital_filter.html:20]: Shouldn't be this section exactly like the section on the aidrequest_filter template?

File: main/templates/main/hospital_filter.html [hunk 3]
```diff
@@ -3,75 +3,76 @@
...
+      {
+          name: "{{ hospital.name }}",
+          town: "{{ hospital.city }}",
+          country: "{{ hospital.country }}",
+          latitude: {{ hospital.latitude|default:0.0|unlocalize }},
+          longitude: {{ hospital.longitude|default:0.0|unlocalize }},
+          text: "{% for aid in hospital.aidrequest_set.all %}{{ aid }}{% endfor %}",
+          link: "{% url 'hospitalfordonor_detail' hospital.id %}",
+          supply_count: "{{ hospital.aidrequest_supply_count }}",
+          repair_count: "{{ hospital.aidrequest_repair_count }}",
```
Comment (fmarani) [inline main/templates/main/hospital_filter.html:72]: We should get rid of the fields that we do not use in the popup. Text, town and country.

File: main/static/js/map.js (hunk 1 above)
Comment (VictorMirandaEE) [reply main/static/js/map.js:104]: This is a work in progress. Next release there should be other improvements/cleanup.

File: main/templates/main/hospital_filter.html (hunk 3 above)
Comment (VictorMirandaEE) [reply main/templates/main/hospital_filter.html:72]: Yes. This is a work in progress. Next release there should be other improvements/cleanup.

File: main/templates/main/hospital_filter.html (hunk 2 above)
Comment (VictorMirandaEE) [reply main/templates/main/hospital_filter.html:20]: No. They are different.

hospital_filter related screen:
https://app.zeplin.io/project/5eb348cc15c3b04b8c301a69/screen/5eb349154c23207639bd7955

aidrequest_filter related screen:
https://app.zeplin.io/project/5eb348cc15c3b04b8c301a69/screen/5eb3491506a63c4b84e5fd9a

**Comment by VictorMirandaEE** [comment]
Please proceed with the PR.

**Pull request merged**